src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/seed/cas1/Cas1AutoScript.kt

```diff
@@ -0,0 +1,98 @@
+
```

Comment:
Did you consider holding  the `data` and `document` JSON structures separately? 

This is what we were doing in CAS2 with separate files for the application data and application document blobs.  When there were changes to the questionnaire it was then quite easy to compare the seed fixture in the API with the UI's integration tests fixture (e.g. [applicationData.json](https://github.com/ministryofjustice/hmpps-community-accommodation-tier-2-ui/blob/b191ff80659ca31a79e9d62a87ea00868aa311c7/integration_tests/fixtures/applicationData.json))

e.g. https://github.com/ministryofjustice/hmpps-approved-premises-api/pull/1318

It might make for a more readable and maintainable approach?

Here's [CAS1 work in progress](https://github.com/ministryofjustice/hmpps-approved-premises-api/pull/1434/files) in which I suggest separate fixture files for:

- application data
- application document
- assessment data
- assessment document
- placement application data
- placement application document

Reply:
That's a great idea, i'll do that before merging. Currently this only pushes the application to pre-submission so we'll only need one blob for now.